common/openstack_release_info

```diff
@@ -24,12 +24,45 @@ declare -A lts=( [trusty]=icehouse
                  [jammy]=yoga
                  [noble]=caracal
 )
+
+# These hardcoded sorted lists replace the previous "sorted" lists
+# because we cannot sort when the alphabet wraps around.
+declare -a lts_releases_sorted=( caracal
+                                 yoga
```

Comment:
The indentation is a bit off.

Reply:
Yoga is still indented but starting from ussuri the indentation is too far left.

Reply:
Maybe an issue with tab vs. space?

Reply:
Thanks, that looks better.